Orange/widgets/widget.py

```diff
@@ -198,32 +197,48 @@ def createPixmapWidget(self, parent, iconName):
         gui.rubber(self.warning_bar)
         self.warning_bar.setVisible(False)
 
-        self.topWidgetPart = gui.widgetBox(self,
-                                           orientation="horizontal", margin=0)
-        self.leftWidgetPart = gui.widgetBox(self.topWidgetPart,
-                                            orientation="vertical", margin=0)
+        self.want_main_area = self.want_graph or self.want_main_area
+
+        class Splitter(QSplitter):
```

Comment:
Is there any reason why each widget has a separate definition of the splitter class? This is useful for getting data from the closure, but these two classes don't use it. What about moving this class from the method into OWWidget's namespace? (I know I'm nitpicking.)